demos/dotnet/dotnet/dotnet.cs

```diff
@@ -43,6 +43,10 @@ static void Main(string[] args)
                 Build(args, Log);
                 break;
 
+            case "/lib":
```

Comment:
this should be /target:library. /lib csc.exe option is to link additional libraries.